sisyphus/load_sharing_facility_engine.py

```diff
@@ -48,6 +48,11 @@ def _system_call_timeout_warn_msg(self, command: Any) -> str:
             return f"SSH command timeout: {command!s}"
         return f"Command timeout: {command!s}"
 
+    def _system_call_error_warn_msg(self, command: Any) -> str:
```

Comment:
Is this maybe a method which could go into `EngineBase`?

Reply:
Maybe, but applying the same train of thought we'd also have to migrate `_system_call_timeout_warn_msg`. Your call.

Reply:
I will resolve this. Please reopen it if you think it's necessary. As stated, I would just leave it for another PR.